### DISCUSSION TITLE
申请退款

### DISCUSSION THREAD

**Discussion by Cranechenug007** [getlantern/lantern-forum-cn#552] [Q&A]
自从买了蓝灯，这个Chatgpt登录就没稳定过，只有SG节点的时候才能上，其他节点都上不去。之前听你们的反馈，说马上自选节点的软件要出来，我信了。到现在还没有出来。我现在已经忍无可忍了。麻烦退款吧。你看需要我提供什么资料？

**Comment by MaxMKMax** [comment]
@Cranechenug007 

很抱歉. 让您失望了. 由于一些限制所以延迟了. 您只需要提供我支付单据和账号邮箱即可.  谢谢!
